src/main/java/uk/gov/ons/census/casesvc/logging/EventLogger.java

```diff
@@ -71,6 +73,30 @@ public void logRefusalEvent(
         eventMetaDataDateTime);
   }
 
+  public void logFulfilmentRequestedEvent(
+      Case caze,
+      UUID caseId,
+      String eventDescription,
+      EventType eventType,
+      FulfilmentRequestDTO payload,
+      EventDTO event) {
+
+    Event loggedEvent = new Event();
```

Comment:
is there a reason we aren't called "logEvent" like we do for the logReceiptEvent and logRefusalEvent methods?